### DISCUSSION TITLE
Wikis for private repository in organization

### DISCUSSION THREAD

**Discussion by vsbel** [community/community#144984] [New to GitHub] [answered]
Hello everyone! I have a problem with adding wikis for my private repository in organization.

My colleague have created an organization and I created private repository in it. After that I tried to add Wikis in settings but saw that I need to upgrade my account or to make my repository public. I read in GitHub Docs that users without Pro-version can't add Wikis in private repositories, so I got a student Pro-version for my account. But I still can't add Wikis, I only see: 

> Upgrade or make this repository public to enable Wikis. Contact the owner of the repository to enable GitHub Wikis.

But I'm an owner of this repository and I have Pro-version. I see in documentation that:
> Wikis are available in public repositories with GitHub Free and GitHub Free for organizations, and in public and private repositories with GitHub Pro, GitHub Team, GitHub Enterprise Cloud and GitHub Enterprise 

So, I have student Pro-version, I'm the member of organization and the owner of private repository in it, but I can't make Wikis enable for this repository. 
What's the problem? Maybe private repositories located in an organizations can't have Wikis?

**Comment by AuroraEchos** [comment, marked as answer]
I understand that you created a private repository within your organization and tried to add wiki pages, but found that even though you had upgraded to the Pro version, you still could not enable wiki pages. According to GitHub's official documentation, the wiki feature can be used for public warehouses as well as for private warehouses owned by individuals, but not for private warehouses owned by organizations. Therefore, the problem you are experiencing may be that the private repository is located within the organization, which does not support enabling wiki pages. To enable wiki pages, you may need to expose the warehouse or move the warehouse to a personal account. Actually, I don't quite understand, but I especially hope to help you. You can also ask other people with development experience.
